packages/back-end/src/models/PastExperimentsModel.ts

```diff
@@ -10,6 +10,7 @@ const pastExperimentsSchema = new mongoose.Schema({
     {
       _id: false,
       trackingKey: String,
+      experimentName: String,
```

Comment:
I think we also want to add a `variationNames` array property so we can use that to pre-fill the Add Experiment form.

Reply:
Resolved by new changes